akka-persistence-query/src/main/scala/akka/persistence/query/typed/EventEnvelope.scala

```diff
@@ -256,6 +256,26 @@ final class EventEnvelope[Event](
     metadata.toJava
   }
 
+  /**
+   * Java API: Drop metadata of the given type if present
+   */
+  def removeMetadata(metadataType: Class[_]): EventEnvelope[Event] =
```

Comment:
Note: No type param here means we can use the same method name.

Reply:
add it to the other EventEnvelope as well, since we still use that in some places and it has metadata